apps/code-examples/src/app/code-examples/lists/filter/modal/filter-modal.component.ts

```diff
@@ -64,13 +64,13 @@ export class FilterModalComponent {
   }
 
   #setFormFilters(appliedFilters: Filter[]): void {
-    for (let i = 0; i < appliedFilters.length; i++) {
-      if (appliedFilters[i].name === 'fruitType') {
-        this.fruitType = `${appliedFilters[i].value}`;
+    for (const appliedFilter of appliedFilters) {
```

Comment:
Iteration loops were updated due to a new eslint rule `@typescript-eslint/prefer-for-of`.